docs/reference_guide.md

```diff
@@ -1709,7 +1709,20 @@ const char *usbcore_name = "usbcore";
 
 ## 9. `uaddr()`: Address resolution, user-level
 
-Syntax: `uaddr(char *name)`
+Syntax: `u64* uaddr(symbol)`
```

Comment:
It doesn't always return a `u64*` now with these changes, right?

Reply:
Nope it depends. Any suggestions for a better way to write it down? I think that having the return type in the `syntax` is useful but when we do it it must be correct

Reply:
@ajor poke

Reply:
Maybe we could list all the forms? e.g.
```
u64 *uaddr(symbol)
u32 *uaddr(symbol)
u16 *uaddr(symbol)
u8 *uaddr(symbol)
```

Reply:
Guess that works, what do you think of the new version @ajor